resources/qml/WelcomePages/StepPanel.qml

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) 2019 Ultimaker B.V.
```

Comment:
People usually call a panel containing several steps a "wizard".

Reply:
Renamed in fd0a60f8dc3205e357e69b34e247e6f06f67b76e